src/modules/units/components/UnitUtilizationByUnitType.tsx

```diff
@@ -0,0 +1,50 @@
+import { Grid, Stack, Typography } from '@mui/material';
+
+import UnitUtilizationChart, {
+  UnitVisualizationChartLegend,
+} from '@/modules/units/components/UnitUtilizationChart';
+import { UnitTypeCapacityFieldsFragment } from '@/types/gqlTypes';
+
+interface Props {
+  unitTypes: UnitTypeCapacityFieldsFragment[];
+  variant?: 'stacked' | 'grid';
+}
+
+// Unit capacity charts grouped by unit type
+const UnitUtilizationByUnitType: React.FC<Props> = ({
+  unitTypes,
+  variant = 'stacked',
+}) => {
+  const hasMultipleUnitTypes = unitTypes.length > 1;
+
+  return (
+    <Stack gap={1}>
+      {unitTypes.map((unitType, idx) =>
+        variant === 'stacked' ? (
+          <Stack gap={0.25}>
```

Comment:
```suggestion
          <Stack key={unitType.id} gap={0.25}>
```
to avoid console warning